### DISCUSSION TITLE
What does the comment for configmap disabled actually mean?

### DISCUSSION THREAD

**Discussion by adepssimius** [authelia/chartrepo#260] [Q&A]
https://github.com/authelia/chartrepo/blob/master/charts/authelia/values.yaml#L412

I would expect `disabled: false` to create a config map and thus not require a persistent volume to store the config, but the note above says
> If this is false you need to provide a volumeMount via PV/PVC or other means that mounts to /config.

I don't understand how to set the setting to use the config map and not use a persistent volume based on the comment and naming.

**Comment by elaurensx** [comment]
`configMap.disabled.false` does indeed create a configMap resource, as you'd expect. 

But this chart version (0.9.5) contains other inconsistencies (f.ex. in templates/deployment.yaml, .Values.pod.securityContext.pod gets passed to the container and .Values.pod.securityContext.container to the pod). I'd submit PR's to fix these, but I've simply decided to go back to an earlier version (0.8.58). The chart has become way too complex for me without giving anything I'd make us of in return. F.ex. no idea why we went from using `enable: true || false` (0.8.58) to `disabled: true || false` (0.9.5). Can't think a good reason for the switch and I'm nitpicking, but now the template files are full of double negatives ( {{- if not .Values.configMap.disabled -}} instead of the simpler {{ if .Values.configMap.enable -}}.
